bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/parser/Converter.java

```diff
@@ -9,14 +9,21 @@
 
 import java.util.Dictionary;
 
+/**
+ * This interfase provides opportunity to use different parsers for converting JsonObject to Dictionary.
+ *
+ * @author Vasil Ilchev - Initial Contribution
+ *
+ */
 public interface Converter {
```

Comment:
So the Converter is yet another service that parses JSON, which is there for just a small very special use case.
Actually, I cannot remember that we ever discussed having JSON within config description parameters. Looking at your sample files, this imho looks rather ugly. If you now thing of other formats (e.g. XML) where you would put such JSON into feels pretty weird.
I'd suggest to discuss whether we really need it that way or if we could come up with a special syntax that suits our needs and which could be processed within the AbstractModuleHandler directly (without requiring any external services).